local_planner/src/nodes/planner_functions.cpp

```diff
@@ -350,7 +351,7 @@ std::pair<float, float> costFunction(const PolarPoint& candidate_polar, float ob
   const float yaw_cost = (1.f - weight) * cost_params.yaw_cost_param * angle_diff * angle_diff;
   const float yaw_to_line_cost = weight * cost_params.yaw_cost_param * angle_diff_to_line * angle_diff_to_line;
   const float pitch_cost =
-      cost_params.pitch_cost_param * (candidate_polar.e - facing_goal.e) * (candidate_polar.e - facing_goal.e);
+      cost_params.pitch_cost_param * (candidate_polar.e - facing_goal.e) * (candidate_polar.e - facing_goal.e) / (0.2 * goal_distance);
```

Comment:
Where does the 0.2 come from?

Reply:
so before the pith cost was independent from the distance to the goal and this resulted in the drone always arriving too high, because the pitch cost was way too high in comparison to avoiding the ground (like 6m below). With this addition the pitch terms starts being lower than before at 5m from the goal, going to zero the closer we get. This allows the drone to actually reach the point. Same here, I don't think we need to tough this number again

Reply:
Thanks for the explanation. I am just a bit uncomfortable of multiplying some number without any explanation and without it being exposed. Could you maybe add a comment on what 0.2 means and why? I think that would be already helpful